Sources/Authentication/Custom/CustomAuthSession.swift

```diff
@@ -1,16 +1,13 @@
 import AuthenticationServices
```

Comment:
Just an observation which I chatted to Joe and Dylan about - the file structure is a little confusing as someone new to this package. For example `CustomAuthSession` should live at same level as `AppAuthSession` (could be in a different directory) as these things implement the same interface. `TokenResponse` sits next to `CustomAuthSession` but is being used by `AppAuthSession` as well. 

It would also be good to document the differences between `AppAuthSession` and `CustomAuthSession` in the Readme. @DylanRJ @mcgarj I'll raise a "tidy up ticket" for these changes - it would be good to bring through before anyone starts to consume this library.

Reply:
@Oliver-Binns ticket raised here for a bit of tidy up in this repo before other teams start to consume it https://govukverify.atlassian.net/browse/DCMAW-7462 Could you pls add any additional thoughts there if you get time?